specification/machinelearning/Azure.AI.ChatProtocol/main.tsp

```diff
@@ -0,0 +1,37 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+
+import "./operations.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Using the AI acronym similar to what was done for OpenAI"
+@doc("Azure APIs for the Azure Chat protocol.")
+@service({
+  title: "Azure ML Chat",
+})
+@useAuth(
+  ApiKeyAuth<ApiKeyLocation.header, "api-key"> | OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit,
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/v2.0/authorize",
+      scopes: [],
```

Comment:
Why is the `scopes` empty? The may fail our authentication because scopes is required.

Reply:
I think we should have a linter rule here. My understanding is that for DataPlane there is no really a standard default for the scopes so assuming {endpoint}/.default may not be reliable at all so having the TSP define the scopes explicitly seems like a good rule to enforce

Reply:
Issue filed https://github.com/microsoft/typespec/issues/2641

Reply:
In this particular case, I think the scope we'll end up using are from one of the services that will implement the contract (as part of the generative story). We also will expose scopes as a knob that customer can control for the bring-your-own-auth scenarios.